src/index.js

```diff
@@ -38,26 +39,36 @@ exports.promisifyGrpc = () => {
   )
 }
 
-// use setCredentials to initialize authenticated grpc connection
-exports.setCredentials = (socketPath, macaroonPath, tlsCertPath) => {
-  var m = fs.readFileSync(macaroonPath)
-  var macaroon = m.toString('hex')
+// use setCredentials to initialize authenticated grpc connection. If both
+exports.setCredentials = (socketAddr, macaroonData, tlsCertData) => {
+  let m
+  let lndCert
+  if (isBase64(macaroonData)) {
+    m = Buffer.from(macaroon, 'base64')
```

Comment:
is this variable `macaroon` defined? I only see `macaroonData`.

Reply:
👍 Good catch